app/models/lettings_log.rb

```diff
@@ -435,13 +435,29 @@ def self.to_csv(user = nil)
     Csv::LettingsLogCsvService.new(user).to_csv
   end
 
+  def validation_beds
+    if needstype == 2
+      0
+    else
+      beds.nil? ? nil : [beds, 4].min
+    end
+  end
+
+  def validation_la
```

Comment:
Does only supported housing have a location set?

Reply:
Yes - for general needs we ask the user the postcode (and infer the la or ask if we can't) directly and add that to the log. For supported housing we ask the user the scheme (and location if there are more than 1 for the scheme) and then need to use the local authority code from that instead.

Reply:
Do we need to do this at all, then? Look at the method `la` that delegates to `location.location_code` if one is set, otherwise uses `la`.

Reply:
Ah you're right, thanks 👍